v2/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/solarwinds/snap-plugin-lib/v2
+
+go 1.13
+
+require (
+	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
```

Comment:
what's that for in lib? @tomekkolo has just struggled to remove that from psutil/aosystem

Reply:
It's due to 
>> go mod why github.com/StackExchange/wmi
# github.com/StackExchange/wmi
github.com/solarwinds/snap-plugin-lib/v2/tutorial/07-proxy/collector/proxy
github.com/shirou/gopsutil/cpu
github.com/StackExchange/wmi

Maybe we should introduce a module for tutorial to widen dependencies

Reply:
https://github.com/solarwinds/snap-plugin-lib/pull/103/commits/f53f2af02b6c17aef1dcadc9fbfaa17c7f720352